app/src/main/java/com/tpp/theperiodpurse/ui/setting/SettingScreenLayout.kt

```diff
@@ -114,9 +139,12 @@ fun SettingScreenLayout(
                )
            }
            Switch(
-               enabled = false,
+               enabled = true,
                checked = appViewModel.getAllowReminders(),
-               onCheckedChange = {appViewModel.toggleAllowReminders()},
+               onCheckedChange = {
+                   appViewModel.toggleAllowReminders()
+                   onToggleClicked(hasNotificationPermission, context, pickedTime, appViewModel)
```

Comment:
It looks like every time I navigate away from the settings page this switch turns off.
Also, when I turn on log symptoms it sends a notification right away instead of notifying at the specified time

Reply:
This is fixed now!